x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -12,7 +12,7 @@ export const DOCS_PREFIX = docLinks.appSearchBase;
 export const ROOT_PATH = '/';
 export const SETUP_GUIDE_PATH = '/setup_guide';
 export const LIBRARY_PATH = '/library';
-export const SETTINGS_PATH = '/settings/account';
+export const SETTINGS_PATH = '/settings';
```

Comment:
Opinionated change - since SMAS doesn't really have the concept of an "account" any longer, I figured this made sense